clients/src/test/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumerTest.java

```diff
@@ -248,6 +248,18 @@ public void testFailOnClosedConsumer() {
         assertEquals("This consumer has already been closed.", res.getMessage());
     }
 
+    @Test
+    public void testFailConstructor() {
```

Comment:
this is covered by `KafkaConsumerTest.testConstructorInvalidMetricReporters`, so could you please remove it?